Lib/test/datetimetester.py

```diff
@@ -1363,6 +1364,13 @@ def test_isocalendar(self):
             d = self.theclass(2010, 1, 4+i)
             self.assertEqual(d.isocalendar(), (2010, 1, i+1))
 
+    def test_isocalendar_named(self):
+        d = self.theclass(2008, 12, 3)
+        t = d.isocalendar()
+        self.assertEqual(t.year, 2008)
+        self.assertEqual(t.week, 49)
+        self.assertEqual(t.weekday, 3)
+
```

Comment:
Great job adding this test!

I suggest also checking equality with:
1. an explicitly created `IsoCalendarDate` instance
2. an equivalent 3-tuple (for backwards compatibility